source/FreeRTOS_ARP.c

```diff
@@ -1,5 +1,5 @@
 /*
- * FreeRTOS+TCP <DEVELOPMENT BRANCH>
+ * FreeRTOS+TCP V4.3.0
```

Comment:
Can we use mechanism to update this during release? Something similar to [this](https://github.com/FreeRTOS/coreMQTT/blob/main/.github/workflows/release.yml#L79-L90)?